src/Web/Rsp.IrasPortal.Web/Features/Modifications/ReviewAllChangesController.cs

```diff
@@ -81,12 +82,31 @@ public Task<IActionResult> SendModificationToSponsor(string projectRecordId, Gui
     }
 
     [HttpPost]
-    public Task<IActionResult> SubmitToRegulator(string projectRecordId, Guid projectModificationId)
+    public Task<IActionResult> SubmitToRegulator(string projectRecordId, Guid projectModificationId, string overallReviewType)
     {
+        // Default to WithRegulator if not set or review required
+        var statusToSet = ModificationStatus.WithRegulator;
+
+        // Evaluate the review type (case-insensitive, null-safe)
+        if (!string.IsNullOrWhiteSpace(overallReviewType))
+        {
+            switch (overallReviewType.Trim().ToLowerInvariant())
+            {
+                case "no review required":
+                    statusToSet = ModificationStatus.Approved;
+                    break;
+
+                default:
+                    statusToSet = ModificationStatus.WithRegulator;
+                    break;
+            }
```

Comment:
Maybe it's time to create constants for ReviewType, ModificationType and Category. Also, please refactor the above to


```suggestion
            statusToSet = overallReviewType switch
            {
                "no review required" => ModificationStatus.Approved,
                 _ = ModificationStatus.WithRegulator
            }
```

Reply:
refactored now